microk8s-resources/actions/common/utils.sh

```diff
@@ -470,7 +470,8 @@ render_csr_conf() {
     if ! [ "$IP_ADDRESSES" == "127.0.0.1" ] && ! [ "$IP_ADDRESSES" == "none" ]
     then
         local ips='' sep=''
-        local -i i=3
+        local ip_count=$(grep -ce '^IP' certs/csr.conf.template)
```

Comment:
I don't think this fixes the issue. In particular, the `IP.$number` lines in csr.conf.template need not be numbered, and also need not be in sequential order.

For example, the above does not work for this csr.conf.template:

```
....
IP.1 = $SOME_IP
IP.2 = $SOME_OTHER_IP
#MOREIPS#
IP.4 = $CUSTOM_IP
```

This is because grep would return 3, thus add lines with `IP.4`, overwriting our custom IP.